Binary files luacom-org/.git/index and luacom/.git/index differ
diff -rup luacom-org/cmake/lua.cmake luacom/cmake/lua.cmake
--- luacom-org/cmake/lua.cmake	2016-01-07 13:09:07.004304800 +0800
+++ luacom/cmake/lua.cmake	2016-01-08 08:37:51.345072900 +0800
@@ -6,9 +6,9 @@
 # For details see the COPYRIGHT file distributed with LuaDist.
 # Please note that the package source code is licensed under its own license.
 
-set ( INSTALL_LMOD ${INSTALL_LIB}/lua
+set ( INSTALL_LMOD ${INSTALL_LIB}/lua/5.1
       CACHE PATH "Directory to install Lua modules." )
-set ( INSTALL_CMOD ${INSTALL_LIB}/lua
+set ( INSTALL_CMOD ${INSTALL_LIB}/lua/5.1
       CACHE PATH "Directory to install Lua binary modules." )
 
 option ( SKIP_LUA_WRAPPER
